shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -40,21 +40,24 @@ - (BOOL)application:(UIApplication*)application
 
 // Returns the key window's rootViewController, if it's a FlutterViewController.
 // Otherwise, returns nil.
-- (FlutterViewController*)rootFlutterViewController {
++ (FlutterViewController*)rootFlutterViewController {
   UIViewController* viewController = [UIApplication sharedApplication].keyWindow.rootViewController;
   if ([viewController isKindOfClass:[FlutterViewController class]]) {
     return (FlutterViewController*)viewController;
   }
   return nil;
 }
 
++ (void)handleStatusBarTouches:(NSSet*)touches withEvent:(UIEvent*)event {
+  if (self.class.rootFlutterViewController != nil) {
```

Comment:
`self` here is the class.
`[self rootFlutterViewController]`
Remove the nil check, just let -handleStatusBarTouches: message nil.

Reply:
Done